content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+{{< warning >}}
+This guide relies heavily on the
+[stable plugin ca certificate](/docs/tasks/security/cert-management/plugin-ca-cert/) feature. Istio
+does not offer an industry standard recommendation for certificate management. Please consult your
+information security team when using any plugin certificate.  The determination of your company
+policies for certificate management is the choice of your company.
+{{< /warning >}}
+
+{{< tip >}}
+This workflow is tested and validated. The Istio authors feel this workflow is suitable for experimentation
+but not production. Like all alpha features, this workflow is subject to change.
+{{< /tip >}}
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable.
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
+    {{< /text >}}
+
+1. Set the name of your desired namespace as an environment variable.
+
+    {{< text bash >}}
+    $ SERVICE_NAMESPACE="<the name of your service namespace>"
+    {{< /text >}}
+
+1. Create a working directory for files generated in this install guide.
+
+    {{< text bash >}}
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"
+    {{< /text >}}
+
+1. Execute the following commands on each cluster in the mesh where a virtual
```

Comment:
Execute only the commands in step 4 on each cluster? Should this `Execute the following commands on each cluster in the mesh` sentence go to the of the section? It applies to the whole set of instructions, doesn't it?

Reply:
This is poorly worded. I will reword to `Execute the following commands on the cluster in the mesh where the virtual machine connection should be ...`

Reply:
Sure, but you also need to change the CLUSTER_NAME and SERVICE_NAMESPACE variables and mk more dirs for each cluster, so you need to do step 1-3 again on each cluster too.

Reply:
This guide doesn't address adding more than one virtual machine to a service mesh. This is represented by the `SERVICE_NAMESPACE` variable. This guide doesn't address adding multiple clusters - the @rcaballeromx guide covers this process.

Do you have any suggestions to fix these shortcomings?

Cheers
-steve

Reply:
Oh, so maybe I'm confused? But, these instructions don't seem to hang together. How many clusters are we installing Istio on ... now it sound like just 1. If so, then the first sentence of the doc should not say anything about multicluster, this is just a single cluster mesh configuration. In step 1 of this section it refers to "your cluster", I'm assuming that is the one and only cluster we're installing Istio on. If so, then what other cluster are you referring to in the "Execute the following command on the cluster in the mesh ...". There is only one == <your cluster> from step 1, right?

Am I missing something here?

Reply:
Istio is installed in one cluster and one cluster only.

I will work to rectify the language around this first problem.

Cheers
-steve